pages/reference/OS/advanced.md

```diff
@@ -95,6 +95,16 @@ dtoverlay=i2c-rtc,ds1307
 dtoverlay=lirc-rpi
 ```
 
+As of Supervisor v16+, the above will translate to:
+
+```
+dtoverlay=i2c-rtc
+dtparam=dc1307
+dtoverlay=lirc-rpi
+```
+
+This modifies each parameter to be on its own line in order to avoid the 80 character line limit imposed by config.txt. It is the recommended method of setting one or more overlays with their own parameters.
```

Comment:
Only a minor point, but https://www.raspberrypi.com/documentation/computers/config_txt.html#dtparam says that the limit is actually 98 characters.